lib/streamlit/caching/cache_utils.py

```diff
@@ -38,11 +42,17 @@
 _LOGGER = get_logger(__name__)
 
 
+@define
```

Comment:
I'd like to get opinions about using new-style attrs vs old-style attrs vs dataclasses